lib/taskana-core/src/test/java/acceptance/task/QueryTasksWithSortingAccTest.java

```diff
@@ -98,6 +98,42 @@ void testSortByTaskIdAsc() {
     }
   }
 
+  @WithAccessId(user = "admin")
+  @Test
+  void testSortByWorkbasketName() {
```

Comment:
Our naming pattern for tests is:

should_ExpectedBehavior_When_StateUnderTest

Example: "should_AddErrorsForInvalidTaskIds_When_BulkForceCompletingTasks()"